api/src/main/resources/application.properties

```diff
@@ -68,6 +68,7 @@ quarkus.arc.exclude-types=io.apicurio.registry.rest.JacksonDateTimeCustomizer
 quarkus.index-dependency.strimzi-api.group-id=io.strimzi
 quarkus.index-dependency.strimzi-api.artifact-id=api
 
+console.work-path=${java.io.tmpdir}
```

Comment:
The operator overrides this to use an emptydir volume in Kube, maintaining a read-only root container filesystem.